### PR TITLE
161505328 image attachments

### PR DESCRIPTION
**Pull request by ArnoutCreemers**
## Code Review

Please consider the following checklist when reviewing this Pull Request.  
More background and details [here](https://github.com/kabisa/kabisa-guide/blob/master/organization/process/code-review/README.md).

* [ ] Does the code **actually solve** the problem it was meant to solve?
* [ ] Is the code covered by **unit tests**? **Integration tests**?
* [ ] Does anything here need **documentation**? (Focus on *why*, not *what.*)
* [ ] Does any of this code deal with **privacy sensitive information** or affects **security**? Ask an additional reviewer.
* [ ] Is the code easy to **understand** and **change** in the future?
* [ ] Is the same code or concept **duplicated**? Find a balance between DRYness and readability.
* [ ] Does the code reasonably adhere to the Kabisa [**coding standards**](https://github.com/kabisa/kabisa-guide/blob/master/organization/process/code-review/README.md)?
* [ ] Be kind.

### CODE REVIEW

File: src/components/upload/ImageUpload.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+export interface Props {
+  onChange: (images: File[]) => void
+}
+
+interface ImageUploadRef {
+  resetState(): void;
+}
+
+export const ImageUpload = React.forwardRef<ImageUploadRef, Props>((props: Props, ref: Ref<ImageUploadRef>) => {
+  const [previews, setPreviews] = useState<any[]>([]);
```
Comment (egonm12) [inline src/components/upload/ImageUpload.tsx:17]: `previews` is typed as `any`

File: src/modules/feed/components/Transaction/Transaction.tsx [hunk 2]
```diff
@@ -33,6 +34,26 @@ function Transaction(props: TransactionProps) {
               </strong>
               {' '}for{' '}<span data-testid="post-message">{props.transaction.message}</span>
             </div>
+            {props.transaction.images.length > 0 && (
```
Comment (egonm12) [inline src/modules/feed/components/Transaction/Transaction.tsx:37]: nitpick: `> 0` is redundant here.

**Review by egonm12 — COMMENTED** [review]
Left 2 small remarks. I didn't run the code locally and I don't have any experience with GraphQL in the frontend but other than that it looks good!

File: src/components/upload/ImageUpload.tsx (hunk 1 above)
Comment (ArnoutCreemers) [reply src/components/upload/ImageUpload.tsx]: good catch, I've created a type for the previews.

**Pull request merged**